ansible/roles/ansible-tvault-contego-extension/tasks/create_filter.yml

```diff
@@ -3,26 +3,41 @@
   template:
      src: trilio.filters.j2
      dest: "{{FILTERS_FILE_PATH_RHEL}}"
-  when: ansible_os_family == "RedHat" 
+  when: ansible_os_family == "RedHat"
 
 - name: create trilio.filters for mount and unmount for Debian
   template:
      src: trilio.filters.j2
      dest: "{{FILTERS_FILE_PATH_DEBIAN}}"
-  when: ansible_os_family == "Debian" 
-  
+  when: ansible_os_family == "Debian"
+
+- name: find location of nova sudoers file
+  shell: find /etc/sudoers.d -name "nova*" | grep -Ew  'nova|nova_sudoers'
+  ignore_errors: yes
+  register: output
+
+- debug:
+    msg: "{{ output.stdout}}"
+  when: output.stdout != ""
+
 - name: Add privsep-helper to nova sudoers file for RHEL
   lineinfile:
-     path: /etc/sudoers.d/nova
+     path: "{{ output.stdout}}"
```

Comment:
Do we need different sections for RadHat/Debian? playbook seems to same for both.

Reply:
For future perspective we added for both Debian and RHEL ,if some changes required on OS basis

Reply:
Requested changes done